Hanna_Chulets/03.GIT.Hosting/03.GIT.Hosting.md

```diff
@@ -0,0 +1,43 @@
+# Git hosting
+
+## Repo URLs 
+
+https://github.com/chulets/GIT-hosting
+
+https://gitlab.com/chulets/gitlab-hosting
+
+https://bitbucket.org/gannulka/bitbucket-hosting
+
+
+## Images with Slack notifications
+
+https://telegra.ph/03-GIT-hosting-06-16
+
+
+## Script
+
+I've decided to add alias to ~/.bashrc
+
+```bash
+
+alias git_push_all='for r in $(git remote); do git push -u $r main; done'
+
+```
+so that every time when I need to push main branch to all remote repos, I'll call a command git_push_all
```

Comment:
Looks like you did changes in repomm your remote has changes which your local repo does not have